backend/app/assets/stylesheets/spree/backend/components/_currency_selector.scss

```diff
@@ -0,0 +1,39 @@
+.currency-selector-symbol {
+  min-width: 2.5em;
+  text-align: center;
+}
+
+.currency-selector {
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  padding-left: .5rem;
+  border: 0;
+  background: transparent;
+
+  -webkit-appearance:none;
+  -moz-appearance:none;
+  appearance:none;
+
+  background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='1024' height='640'><path d='M1017 68L541 626q-11 12-26 12t-26-12L13 68Q-3 49 6 24.5T39 0h952q24 0 33 24.5t-7 43.5z'></path></svg>") 90%/12px 6px no-repeat;
+
+  font-family: inherit;
+  color: inherit;
+
+  &:-moz-focusring {
+    color: transparent;
+    text-shadow: 0 0 0 #000;
+  }
+}
```

Comment:
:+1: 

I think I'm going to try `&:hover:not(:active)`. When testing with just `:hover` it felt a little weird. When opening the select, the cursor changed from `pointer` to `arrow` and then quickly back. Using `:hover:not(:active)`, it just switches to `arrow`, and stays that way while open. This is the same behaviour we get from the select2 controlled selects.